src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/MicrosoftIdentityPlatform/MicrosoftIdentityPlatformApplicationManager.cs

```diff
@@ -241,11 +242,20 @@ public class MicrosoftIdentityPlatformApplicationManager
                     updatedApp.Web.ImplicitGrantSettings.EnableAccessTokenIssuance = toolOptions.EnableAccessToken.Value;
                 }
 
-                var idTokenEnabled = existingApplication.Web.ImplicitGrantSettings.EnableIdTokenIssuance ?? false;
-                if (!idTokenEnabled)
+                if (requestFromVS)
```

Comment:
@vijayrkn Added a flag to determine whether it was a tooling request or a CLI request

Reply:
I don't quite understand. Isn't update application called from both commandline & VS?

Reply:
Oh, I see now, that's my misunderstanding.

Reply:
I can add an ID token parameter to the arguments and have VS include it in the command line arguments